docs/cli/miactl/30_commands.md

```diff
@@ -79,7 +80,7 @@ miactl context list
 The `context auth` subcommand allows you to setup the Console Service Account you want to use to authenticate to the Console.
 
 ```sh
-miactl context auth [flags]
+miactl context auth CONTEXT [flags]
```

Comment:
The [same comment](https://github.com/mia-platform/documentation/pull/1219#discussion_r1515809630) from !1219 applies, can you move this change to the miactl repo?

the one under version_docs can be kept here instead.

Thanks